docker/trustlines/README.md

```diff
@@ -31,10 +31,17 @@ We need to do some initial setup and configuration for the system to work. You
 need to provide the `addresses.json` file, which should be put in the directory
 alongside the `docker-compose.yml` file.
 
-This directory contains working example files for contracts already
-deployed on the laika blockchain. If you deploy your own contracts,
-please adapt `addresses.json` accordingly.
+You can find some prepared files for each network in the `config/` directory at
+the root of this repository. E.g. if you want to connect to the Trustlines
+Blockchain and index all registered currency networks of the foundation, you can
+copy it like that:
 
+```sh
+$ cp ../../config/addresses_tlbc.json ./addresses.json
+```
+
+This also includes the references to the identity related contracts. If you
+deploy your own contracts, please adapt `addresses.json` accordingly.
```

Comment:
I read this first like you could deploy your own identity contracts (which you can), but that's not necessary and will make the address recovery not work. Therefore please replace with:

If you deploy your own currency networks,... accordingly.

Reply:
:heavy_check_mark: